packages/material-ui/src/AvatarGroup/AvatarGroup.test.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { expect } from 'chai';
 import { getClasses, createMount, describeConformance, createClientRender } from 'test/utils';
-import Avatar from '@material-ui/core/Avatar';
+import Avatar from '../Avatar';
```

Comment:
We have been talking with Sebastian about keeping the global import in the test to be as much end to end as possible. 
```suggestion
import Avatar from '@material-ui/core/Avatar';
```